Add R to the list of packages?

Is there any way to install [R](https://www.r-project.org/) with Chromebrew? Thanks! :)

@bfayers thanks for the link. I've downloaded the file successfully but this command doesn't seem to work `./configure --prefix=$HOME/R`. I've tried different directories (`$HOME`, `$HOME/R/bin` and `$HOME/bin`) but I keep getting this error: `configure: error: No F77 compiler found`. Any suggestions?

Running it with `sudo` gives me the following error: 

`checking for gcc... gcc
checking whether the C compiler works... no
configure: error: in `/usr/local/bin/R-3.3.1':
configure: error: C compiler cannot create executables
See `config.log' for more details'

--- a/packages/r.rb
+++ b/packages/r.rb
@@ -1,0 +1,17 @@
+require 'package'
+
+class R < Package
+  version '3.3.2'
+  source_url 'https://cran.rstudio.com/src/base/R-3/R-3.3.2.tar.gz'
+  source_sha1 '0e39e9c2d28fe6bab9c55ca23e08ba8727fd2fca'
+
+  def self.build
+    system './configure',
+      '--with-x=no' # X is not available
+    system 'make'
+  end
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+  end
+end

